datacite-commons/README.md

```diff
@@ -0,0 +1,11 @@
+# datacite-commons
+
+Provides a handler to allow reservation of DOIs (non-findable DOIs for publications with status "Draft")
+
+## Why does this module exist?
+
+This is a short-circuit of a process, which would otherwise be handled as an asynchronous process.  This is an unfortunate optimization for a frontend design that had timing that was considered unacceptable by the end users.
```

Comment:
Bit difficult to parse without historical context. Perhaps simply rewrite to something like (assuming I understand correctly):

> This is a workaround that short-circuits the usual asynchronous process for generating DOIs. This is necessary to display new DOIs to end users within an acceptable time-frame.

Reply:
Re-wrote this to match what was intended based on what you commented.